### ISSUE TITLE
Creating aliases for an MDX file

### ISSUE THREAD

**Issue by BrettJay** [mathieudutour/gatsby-digital-garden#30]
Really cool project!

I was having a poke through the `gatsby-transformer-markdown-references` package, and I notice there seems to be some code for an [aliases frontmatter object](https://github.com/mathieudutour/gatsby-digital-garden/blob/master/packages/gatsby-transformer-markdown-references/src/on-create-node.ts#L21-L33).

Would it be possible for gatsby-digital-garden to make use of this aliases object to allow you to define alternative aliases in a page...

```
---
title: Hello World
aliases: [ 'hello world', 'alternative alias' ]
---

Lorem ipsum
```

..and then reference one of the alternative aliases in another page?

```
---
title: Another page
---
Check out [[alternative alias]].
```

**Comment by mathieudutour** [comment]
Yeah I have some code for that but the references are generated at a time where the frontmatter isn't loaded yet. So I need to find a solution for that
